frontend/app/src/pages/main/managed-workers/create/components/create-worker-form.tsx

```diff
@@ -320,6 +343,7 @@ export default function CreateWorkerForm({
 
   const [envVars, setEnvVars] = useState<KeyValueType[]>([]);
   const [isIac, setIsIac] = useState(false);
+  const [hasAutoscaling, setHasAutoscaling] = useState(false);
```

Comment:
I think it might be a good idea to do something like this instead:

```typescript
type ScalingType = "Autoscaling" | "Static"
const scalingTypes: ScalingType[] = ["Autoscaling", "Static"]

...

const [scalingType, setScalingType] = useState<ScalingType>("static");
```

Reply:
Implementing this

Reply:
@mrkaye97 done 👍